### PR TITLE
237024_Add constraint to only generate statement for active schemes

### PR DESCRIPTION
**Pull request by AbidemiAdio**

### CODE REVIEW

**Review by GodsonLeigh — COMMENTED** [review]
Looks good, just one question relating to two lines of code commented out in the test file

**Review by GodsonLeigh — DISMISSED** [review]
Fantastic, thanks!

File: app/processing/schedule/get-scheduled-settlements.js [hunk 1]
```diff
@@ -10,19 +9,19 @@ const getScheduledSettlements = async (started, transaction) => {
...
       attributes: []
     }],
+    attributes: [
+      'scheduleId',
+      'settlementId'
+    ],
     where: {
       category: STATEMENT,
       completed: null,
-      '$settlements.sourceSystem$': SOURCE_SYSTEM.SFI,
```
Comment (AbidemiAdio) [inline app/processing/schedule/get-scheduled-settlements.js:25]: This line of code is removed as statement generation is not limited to SFI scheme but any scheme that has been set as active and current in the database.

Comment (GodsonLeigh) [reply app/processing/schedule/get-scheduled-settlements.js:25]: Thanks for the context

**Review by GodsonLeigh — APPROVED** [review]
:)

**Pull request merged**